packages/terra-table/src/subcomponents/ColumnHeaderCell.jsx

```diff
@@ -424,7 +424,7 @@ const ColumnHeaderCell = (props) => {
       onMouseDown={isSelectable && onColumnSelect ? handleMouseDown : undefined}
       onKeyDown={(isSelectable || isResizable) ? handleKeyDown : undefined}
           // eslint-disable-next-line react/forbid-component-props
-      style={{ width: `${width}px`, height: isActionCell ? 'auto' : headerHeight, left: cellLeftEdge }}
+      style={{ height: isActionCell ? 'auto' : headerHeight, left: cellLeftEdge, top: isActionCell ? headerHeight : undefined }}
```

Comment:
Awesome job, very nice solution, very neat! 

One small suggestion. To avoid having `top: undefined` (see snapshots), I suggest adding `style.top` conditionally, only when it is an action cell. For that in line 405 create a variable like:

```
const cellStyle = {
    height: isActionCell ? 'auto' : headerHeight,
    left: cellLeftEdge,
  };

  if (isActionCell) {
    cellStyle.top = headerHeight;
  }
```
  Then in line 427 assign to that variable:
`  style={cellStyle}`

Reply:
I value of undefined or null will make the attribute be ignored.  Do we need the additional logic in that case?

Reply:
Let me double check this one, but my understanding was that for attribute it will be ignored, but for css property it'l be populated with "undefined". But now I am double questioning myself, so let me check.

Reply:
You were right, it is being disregarded. I guess I mixed it up with direct assigning css properties via js. Approving!